lib/bootscale/file_storage.rb

```diff
@@ -13,11 +13,13 @@ def load(load_path)
 
     def dump(load_path, cache)
       path = cache_path(load_path)
-      return if File.exist?(path)
+      raise "This should not happen" if File.exist?(path)
```

Comment:
true true ... I'll change the test to 'it ca dump twice' or jut drop it ?

On Mon, Aug 24, 2015 at 4:51 PM, Jean Boussier <notifications@github.com>
wrote:

> In lib/bootscale/file_storage.rb
> <https://github.com/byroot/bootscale/pull/16#discussion_r37818959>:
>
> > @@ -13,11 +13,13 @@ def load(load_path)
> >
> >      def dump(load_path, cache)
> >        path = cache_path(load_path)
> > -      return if File.exist?(path)
> > +      raise "This should not happen" if File.exist?(path)
>
> Yes, but I think I would simply remove this line. If it's never gonna
> happen anymore, we might as well not do a useless stat call :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/byroot/bootscale/pull/16/files#r37818959>.
>